src/README.md

```
@@ -0,0 +1,305 @@
+* Hot Deploy
+* Debugging
+
+The debug port can be specified with the environment variable `DEBUG_PORT`. `DEBUG_PORT` is only valid if `DEV_MODE=true`.
+
+A simple example command for running the container in development mode is:
+```
+podman run --env DEV_MODE=true my-image-id
+```
+
+To run the container in development mode with a debug port of 5454, run:
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)